tutorials/gui_config.md

```diff
@@ -15,10 +15,34 @@ There are a few places where the GUI configuration can come from:
 2. A `<gui>` element inside an SDF file
 3. The default configuration file at `$HOME/.ignition/gazebo/gui.config` \*
 
-Each of the items above takes precedence over the ones below it. For example,
-if a user chooses a `--gui-config`, the SDF's `<gui>` element is ignored. And
-the default configuration file is only loaded if no configuration is passed
-through the command line or the SDF file.
+Each of the items above takes precedence over the ones below it by default.
+For example, if a user chooses a `--gui-config`, the SDF's `<gui>` element is
+ignored. And the default configuration file is only loaded if no configuration
+is passed through the command line or the SDF file.
+
+The default behaviour can be overridden using the `--gui-config-options`
+command line argument, which offers options:
+
+* `force`: Force the use of the GUI config file, ignoring the SDF file. If a
+  `--gui-config` is provided explicitly, that is used, otherwise, the default
+  config file is used.
+* `ignore`: Ignore the config file completely and load only plugins from SDF.
+* `prepend`: Load plugins from both the config file and the SDF file. The config
```

Comment:
The only thing that I think could be troublesome here is if you had duplicate plugins.  The scenario that I'm thinking of would be  having "specific" configuration in the SDF file and "generic" configuration in the `gui.config` file.

In that case, I think that you would want to prefer the "specific" configuration from the SDF file.

The implementation should take into account this scenario as well as the possibility of having duplicated plugins from the two configurations available.

Reply:
Yeah that's complicated to handle, because there are valid use cases for duplicate plugins, on both the GUI and the server. You may want multiple `ImageDisplay`s or multiple `WindEffects`, for example.

I don't have a good idea on how to handle that. Maybe each plugin can be responsible for shortcutting multiple instantiation?

Reply:
I added a `merge` option to cover this use case, with an example. Let me know what you think 4ab37d6dddc2d9feb64a2baaed21719a7b95ae88. Coming back to this after so many months, now I think this will be my favorite option.

Reply:
I reduced the number of options to 2 in d7a7677c3791890d5116705ecbc1f06a682b7044 and will work on the implementation now